Draft review (#59)
astropy/astropy-v5.0-paper: src/ms.tex

\subsubsection{Data Central and the Anglo-Australian Telescope Archive}
% \secauthor{James Tocknell}
Data Central is one of the five Australian Virtual Observatory\footnote{Known as
  the All-Sky Virtual Observatory, or ASVO.} nodes, providing both Australian and
international astronomers access to and services for surveys of the southern
sky. Whilst primarily hosting optical data, including the Anglo-Australian
Telescope (AAT) Archive\footnote{The AAT Archive is currently jointly managed by
  Data Central, Astronomy Australia Limited (AAL), and National Computational
  Infrastructure (NCI).}, Data Central also assists theoretical and radio
astronomers with hosting and managing their data.

\python is widely used by Data Central, and \astropy and its wider ecosystem
play a vital role in the initial ingestion, reduction, access and visualization

> of the hosted data and services. For example, the AAT Archive hosts date back to

Reviewer: I think this was right as is! Slight rephrasing to make it clearer
```suggestion
of the hosted data and services. For example, the AAT Archive hosts data from as early as
```
